lib/feed.js

```diff
@@ -263,6 +265,15 @@ Feed.prototype.query = function query_feed() {
       return self.retry();
     }
 
+    if(resp.statusCode == 404) {
+        destroy_response(resp)
+        self.log.warn('Database not found. Stopping changes feed after seq '+self.since)
```

Comment:
Sure, I'll change. I saw this format in a few places and assumed it was the norm.